src/open_inwoner/pdc/models/product.py

```diff
@@ -101,6 +101,15 @@ class Product(models.Model):
             "Product content with build-in WYSIWYG editor. By adding '[CTABUTTON]' you can embed a cta-button for linking to the defined form or link"
         ),
     )
+    video = models.ForeignKey(
```

Comment:
Is this what we want? A `ForeignKey` from `A` to `B` defines a many-to-one relationship. I this case, many products can be associated with a single video (as in: one video covers different products), but each product can have only one video. My first thought is that, if anything, it should be the other way around (you could have several videos for a single product, highlighting different aspects perhaps), or perhaps even have  a many-to-many relationship.

Reply:
We specifically went for this because it was the simplest solution, until ultimately we convert the pages to be CMS pages (where the editor can add and positions videos freely).